subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformExecutionBuildOperationIntegrationTest.groovy

```diff
@@ -97,10 +106,32 @@ class ArtifactTransformExecutionBuildOperationIntegrationTest extends AbstractIn
         def projectTransformIdentification = executionIdentifications.find { it.inputArtifactName == 'producer.jar' }
         def externalTransformIdentification = executionIdentifications.find { it.inputArtifactName == 'test-4.2.jar' }
 
+        [projectTransformIdentification, externalTransformIdentification].each {
+            with(it) {
+                identity != null
```

Comment:
❔  Do these assertions work without `assert` within the `each` block?

Reply:
Yes, I tried. The `with` from Spock seems to be the magic sauce.